### ISSUE TITLE
Problems downloading [ManaMoa]

### ISSUE THREAD

**Issue by Dgraid12** [fmd-project-team/FMD#831]
![imagen](https://user-images.githubusercontent.com/53473707/75315406-13fdc600-5839-11ea-8277-88f730245603.png)

as seen in the image, some do download and others do not .. there are very few that work

**Comment by Tmp341** [comment]
Have you checked if those files are there? They might have been removed etc.

**Comment by Slasar41** [comment]
It's because some images protected behind another DDoS protection and FMD need to support it.
For workaround, open [this link](https://filecdn.xyz//upload/246cdf4d98bda7352c1421defe71f7fd.jpg) and then get the cookies and put it to FMD.

**Comment by Tmp341** [comment]
Have you tried low settings?

Not sure exactly, but CloudFlare likes ddos protection ban. 
Try setting these to ``1`` :
```lua
Enabled=Ticked (True)
MaxTaskLimit=1
MaxThreadPerTaskLimit=1
MaxConnectionLimit=1
UpdateListNumberOfThread=1
```

**Comment by Slasar41** [comment]
@Tmp341 It's not Cloudflare but another ddos protection service.
![image](https://user-images.githubusercontent.com/28888646/75544482-ab8c3000-5a56-11ea-8dd3-036a97165eb4.png)

**Comment by Tmp341** [comment]
What? I saw CF cookies tho. 👀

**Comment by Slasar41** [comment]
> ~What? I saw CF cookies tho. 👀~
> 
> Oh wait, you mean filecdn is behind ddos-guard.

Yes, some images place behind DDoS-GUARD and FMD need to support it to get the cookies automatically.
